course/models.py

```diff
@@ -474,6 +474,14 @@ def clean(self):
     def is_valid_language(self, lang):
         return lang == "" or lang in [key for key,name in settings.LANGUAGES]
 
+    @property
+    def default_language(self):
+        language = self.language
+        language_list = [x for x in language.split("|") if x]
```

Comment:
Just `language.lstrip('|').split('|', 1)[0]` might be nice. No need to build a list

```
>>> timeit("foo = [x for x in l.split('|') if x]", "l = '|en|fi|'")
0.2993435220005267
>>> timeit("foo = [x for x in l.split('|') if x]", "l = 'fi'")
0.20665564800037828
>>> timeit("foo = l.lstrip('|').split('|', 1)[0]", "l = '|en|fi|'")
0.17270740999992995
>>> timeit("foo = l.lstrip('|').split('|', 1)[0]", "l = 'fi'")
0.10890936499981763
```